host.go

```diff
@@ -500,6 +500,14 @@ func (h *gpbftRunner) BroadcastMessage(ctx context.Context, msg *gpbft.GMessage)
 		return fmt.Errorf("encoding GMessage for broadcast: %w", err)
 	}
 
+	_, err = h.participant.ValidateMessage(msg)
+	if err != nil {
+		log.Errorf("full message validation failed: %+v", err)
+	}
+	_, err = h.pmv.PartiallyValidateMessage(pmsg)
+	if err != nil {
+		log.Errorf("partial message validation failed: %+v", err)
+	}
```

Comment:
We effectively get this from the pubsub right? in that all messages are validated by self before being published. The only thing is the footprint of this which should be minimal anyhow.

Double checking the above is more or less the train of thought here.

Reply:
ahh, this should have been deleted, it is leftover debugging code